components/processor/gcomapiprocessor/config.go

```diff
@@ -23,6 +23,7 @@ type clientConfig struct {
 type cacheConfig struct {
 	CompleteRefreshDuration    time.Duration `mapstructure:"complete_refresh_duration"`
 	IncrementalRefreshDuration time.Duration `mapstructure:"incremental_refresh_duration"`
+	GrafanaClusterFilters      string        `mapstructure:"grafana_cluster_filters"`
```

Comment:
maybe we should not put it as part of the cache config even though it is implemented as part of it, can be put in on the same level as client and cache?

Reply:
agree! fixed